packages/did-provider-jwk/src/jwk-did-provider.ts

```diff
@@ -23,7 +23,7 @@ export class JwkDIDProvider extends AbstractIdentifierProvider {
   async createIdentifier(args: ICreateIdentifierArgs, context: IRequiredContext): Promise<Omit<IIdentifier, 'provider'>> {
     const key = await importProvidedOrGeneratedKey(
       {
-        kms: args.kms ?? this.defaultKms,
+        kms: args.kms ?? this.defaultKms ?? '',
```

Comment:
Rather have a ts-ignore here. Our KMS handles it correctly when no KMS is provided. Using an empty string will result in errors